pyproject.toml

```diff
@@ -105,12 +105,11 @@ dev = [
     "pytest-cov",
     # For doctest
     "pytest-doctestplus",
-    # For test parameterization
-    "parameterized>=0.9",
     # pytest and its plugins
     "pytest>=6",
     "pytest-xdist",
     "pytest-mock",
+    "pytest-subtests",
```

Comment:
Could you add this to the installation guide? Here: https://docs.pybamm.org/en/latest/source/user_guide/installation/index.html#optional-dependencies

Reply:
Let me add this in another commit

Reply:
Done